DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -507,9 +520,43 @@ bool autoRefresh
                         CompleteWithinMonths = @completeWithinMonths,
                         ValidityMonths = @validityMonths,
                         Mandatory = @mandatory,
-                        AutoRefresh = @autoRefresh
+                        AutoRefresh = @autoRefresh,
+                        RefreshToCustomisationID = @refreshToCustomisationId,
+                        AutoRefreshMonths = @autoRefreshMonths,
+                        ApplyLpDefaultsToSelfEnrol = @applyLpDefaultsToSelfEnrol
                     WHERE CustomisationID = @customisationId",
-                new { completeWithinMonths, validityMonths, mandatory, autoRefresh, customisationId }
+                new
+                {
+                    completeWithinMonths,
+                    validityMonths,
+                    mandatory,
+                    autoRefresh,
+                    customisationId,
+                    refreshToCustomisationId,
+                    autoRefreshMonths,
+                    applyLpDefaultsToSelfEnrol,
+                }
+            );
+        }
+
+        public IEnumerable<Course> GetCentrallyManagedAndCentreCourses(int centreId, int? categoryId)

```

Comment:
Is this method meant to be equivalent to GetCoursesAvailableToCentreByCategory (which was renamed at some point)?

Reply:
Oh right, they look almost identical. The only difference I see is that GetCoursesAvailableToCentreByCategory returns inactive courses as well, which I assume we don't want in this case. Should I use this method instead and filter out the inactive courses in the service?

Reply:
I would use the existing method and filter in the service if that is the only difference.

Reply:
Cool, I've removed the GetCentrallyManagedAndCentreCourses data service and just used the existing one.